Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -644,39 +654,40 @@ IPCCommandResult ES::ExportContentBegin(Context& context, const IOCtlVRequest& r
 
 ReturnCode ES::ExportContentData(Context& context, u32 content_fd, u8* data, u32 data_size)
 {
-  const auto iterator = context.title_export.contents.find(content_fd);
-  if (!context.title_export.valid || iterator == context.title_export.contents.end() ||
-      iterator->second.content.m_position >= iterator->second.content.m_content.size)
+  ReturnCode ret = ES_EINVAL;
```

Comment:
I'm probably missing something, but this doesn't seem to error if `position >= size` anymore? Is that intentional?